### ISSUE TITLE
Playback progress (bookmark) sync support

### ISSUE THREAD

**Issue by dbuvid** [memen45/SubMusic#31]
I mainly listen to podcasts while running and I'd love to have a way to sync playback progress for longer episodes between my watch and other devices.

As I understand it the Subsonic API already has support for bookmarking so I was wondering if it would be possible to leverage that to sync playback progress when syncing the watch with a phone.

**Comment by memen45** [comment]
Yes, thanks for opening the issue! Currently I don't have time and the server I currently use as backend (Nextcloud with owncloud/music) does not support bookmarks for podcasts yet. Therefore implementation is currently difficult, but this is indeed an important feature!

**Comment by sixgeek** [comment]
I just came here to underline the importance of that feature. I would even change my beloved podcast app on my Android phone besides purchasing finally a Garmin watch if it's isle to v sync the progress of the playlist\episode bookmark feature would exist.

I'm listening to a lot of podcasts (>24h) during a week and for running I'm using currently only my phone (tracking and podcast).

That feature would be superb 👍

**Comment by memen45** [comment]
Thanks for showing your interest in the feature. In order to provide a little more context, and for future reference:

- [x] bookmark support Subsonic API
- [x] bookmark support Ampache API (V5) ampache/ampache#2591
- [x] bookmark support owncloud/music (with Subsonic) owncloud/music#919
- [ ] bookmark support owncloud/music (with Ampache4)
- [ ] bookmark support SubMusic

Current SubMusic Ampache API support: V4
Current owncloud/music Ampache API support: V4 owncloud/music#909

Ampache API supports bookmarks since v5. Backward compatibility with v4 and v3 was added from v5.2+ onwards. Since the owncloud/music app is used a lot with SubMusic, Ampache API V5 cannot be implemented until owncloud/music decides to implement Ampache5 as well. I will check if there is any update on this!
